tsconfig.json

```diff
@@ -1,35 +1,30 @@
 {
   "compilerOptions": {
+    "emitDeclarationOnly": true,
     "strictNullChecks": true,
     "baseUrl": ".",
-    "paths": {
-      "types": ["./types"],
-      "next-auth": ["./src/server"],
-      "next-auth/adapters": ["./src/adapters"],
-      "next-auth/react": ["./src/client/react"],
-      "next-auth/jwt": ["./src/lib/jwt"],
-      "next-auth/providers/*": ["./src/providers/*"],
-    },
-    "target": "es5",
+    "outDir": ".",
+    "target": "es2019",
     "lib": ["dom", "dom.iterable", "esnext"],
     "allowJs": true,
-    "skipLibCheck": true,
     "strict": false,
```

Comment:
I wonder if we can switch to `strict: true` and rely on `// @ts-expect-error` for the errors we don't want to fix just yet? 💭